src/ast.ml

```diff
@@ -232,7 +232,7 @@ and print_expr fmt (_loc, expr) =
   | Eunop (uop, e) ->
     print_unop fmt uop;
     print_expr fmt e
-  | Ebinop (bop, e1, e2) ->
```

Comment:
So that they're in the same order than what's written in the lua file. And it allows the parser to be a little bit shorter on this specific line.